x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/__snapshots__/generate_latest_transform.test.ts.snap

```diff
@@ -47,6 +47,18 @@ Object {
           "field": "@timestamp",
         },
       },
+      "entity.identity.event.category": Object {
+        "terms": Object {
+          "field": "event.category",
+          "size": 1,
+        },
+      },
+      "entity.identity.log.logger": Object {
+        "terms": Object {
+          "field": "log.logger",
+          "size": 1,
+        },
+      },
```

Comment:
Since these are single value fields, I wonder if we shouldn't have used `top_metric` and then used the same `set` processor as the history?